googlemaps/googlemaps.php

```diff
@@ -23,20 +25,20 @@ function googlemaps_uninstall()
 	logger("removed googlemaps");
 }
 
-function googlemaps_location($a, &$item)
+function googlemaps_location(App $a, array &$item)
 {
 
-	if(! (strlen($item['location']) || strlen($item['coord']))) {
+	if (! (strlen($item['location']) || strlen($item['coord']))) {
```

Comment:
This should be `if (empty($item['location']) || empty($item['coord'])) {`

Reply:
Okay, when I'm back from work.